jsa/features/people.feature

```diff
@@ -12,13 +12,13 @@ Feature: People
     And I should see the "People" heading in the "First sidebar" region
     And I should see "This is your Why I Teach block" in the "First sidebar" region
 
-  @content @live
+  @content @deploy
   Scenario: Faculty node
     Given I am on "people/faculty"
     And I click "Jacob Smith" in the "Content Body" region
     Then I should see "Professor of English" in the "Content Body" region
 
-  @content @live
+  @content @deploy
```

Comment:
You're looking at the wrong scenario. The tags on line 21 refer to the "Faculty taxonomy filter" Scenario.